tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -1273,6 +1273,58 @@ def get_running_jobs():
         time.sleep(1)
 
 
+@pytest.mark.run(order=18)
```

Comment:
How are you selecting the order here? It doesn't seem to fit in with the rest of the tests. My original idea with the order was something like

10-19 (one set of tests), but maybe only 10-14 in use
20-29 (another set of tests
etc...

Currently it doesnt seem like the order is used consistently though

Reply:
I will move it. I just had top my test up, because after this
```
@pytest.mark.run(order=30)
def test_large_volume_register_feature_set_success(client):
```
nothing succeed. It fill queue for minutes ahead

Reply:
done